### PR TITLE
Added Technical Writing to Roles in Tech

### PR DESCRIPTION
**Pull request by pydevsg**
A short description about  the career in the field of Technical Writing.
Fixed Issue - #671 .

### CODE REVIEW

**Comment by pydevsg** [comment]
The slide is yet to complete and I am working on it . If there are certain issue and edits please do refer so as to add/delete them in the ppt.

File: roles-in-tech/technical-writing.md [hunk 1]
```diff
@@ -16,75 +12,56 @@ Here are links to lessons that should be completed before this lesson:
 
 ### Motivation
 
-Here's why this topic should be learned. Strengthen this section by including a company that uses this skillset, and a screenshot example if possible.
+- Being a technical writer allows you to be a first user of the application. So, if you like the experience of using something brand new and playing around with it, fail with it  and improving it, you would love technical writing
```
Comment (alodahl) [inline roles-in-tech/technical-writing.md]: Fail -> failing

Comment (alodahl) [reply roles-in-tech/technical-writing.md]: Remove “So”

File: roles-in-tech/technical-writing.md [hunk 2]
```diff
@@ -16,75 +12,56 @@ Here are links to lessons that should be completed before this lesson:
...
+- Standardize content across platforms and media.
 
 ### Materials
 
-- [Example video (10 min)](https://example.com) - Write a very short description of this resource.
-- [Other example article(20 min read)](https://otherexample.com) - Answer the 3 questions at the end.
-(Be sure to explain how to use these materials in directions in one of the sections below (ie: lesson, guided practice).  If you are not assigning them somehow, they should be placed under Supplemental Materials at the bottom.) 
-
+Technical writers, also called *technical communicators*, prepare instruction manuals, how-to guide journal articles and other supporting documents to communicate complex and technical information more easily. They also develop, gather, and disseminate technical information through an organization's communications channels. 
+- *For better understanding in **Technical Writing** one should refer to this silde* [Roles in Tech: Technical Writing](https://docs.google.com/presentation/d/1p7fa20o7lRyvXuXhsFz8MfHktY5MaqSPSrLdNjocNcE/edit?usp=sharing)
```
Comment (alodahl) [inline roles-in-tech/technical-writing.md]: Silde-> slide

File: roles-in-tech/technical-writing.md [hunk 3]
```diff
@@ -16,75 +12,56 @@ Here are links to lessons that should be completed before this lesson:
...
-
-### Guided Practice
-
-Have the apprentices work with you as you do something step-by-step.  This can also be fulfilled by a detailed tutorial intended for beginners.
+*Here are some of the common mistakes Technical Writers do-*  
+- **Messy Structure** : Many technical documents confuse readers and fail to achieve their aims because they were not planned properly to begin with.
+- **Poor punctuation** : All writers have a passing knowledge of the main set of punctuation marks. Very few, however, outside of professional authors and editors, have a thorough grasp of how each one works. 
+- **Too much abstraction** : People writing in a formal or semi-formal context often go overboard in an effort to make their prose sound proper and elevated.
 
+*Few misconceptions regarding Technical Writing are as follows-*
```
Comment (alodahl) [inline roles-in-tech/technical-writing.md]: Few -> A few

File: roles-in-tech/technical-writing.md [hunk 4]
```diff
@@ -16,75 +12,56 @@ Here are links to lessons that should be completed before this lesson:
...
+*Here are few practical and independent ways to independently learn the skill of Technical Writing*
+- Edit someone else's work.
+- Write a style guide.
+- Read other's technical writing attentively.
 
 ### Challenge
 
-Apprentices can try to do this other thing. Ideally, they will be challenged to connect what they've learned to some previous knowledge or additional research.
-
+- Writing own blogs.
```
Comment (alodahl) [inline roles-in-tech/technical-writing.md]: Make these commands.
Writing -> write

File: roles-in-tech/technical-writing.md (hunk 1 above)
Comment (pydevsg) [reply roles-in-tech/technical-writing.md]: Done

File: roles-in-tech/technical-writing.md [hunk 5]
```diff
@@ -16,75 +12,56 @@ Here are links to lessons that should be completed before this lesson:
...
+- Standardize content across platforms and media.
 
 ### Materials
 
-- [Example video (10 min)](https://example.com) - Write a very short description of this resource.
-- [Other example article(20 min read)](https://otherexample.com) - Answer the 3 questions at the end.
-(Be sure to explain how to use these materials in directions in one of the sections below (ie: lesson, guided practice).  If you are not assigning them somehow, they should be placed under Supplemental Materials at the bottom.) 
-
+Technical writers, also called *technical communicators*, prepare instruction manuals, how-to guide journal articles and other supporting documents to communicate complex and technical information more easily. They also develop, gather, and disseminate technical information through an organization's communications channels. 
+- *For better understanding in **Technical Writing** one should refer to this Silde* [Roles in Tech: Technical Writing](https://docs.google.com/presentation/d/1p7fa20o7lRyvXuXhsFz8MfHktY5MaqSPSrLdNjocNcE/edit?usp=sharing)
```
Comment (alodahl) [inline roles-in-tech/technical-writing.md]: This Silde-> these slides

File: roles-in-tech/technical-writing.md [hunk 6]
```diff
@@ -16,75 +12,56 @@ Here are links to lessons that should be completed before this lesson:
...
+*Here are few practical and independent ways to independently learn the skill of Technical Writing*
+- Edit someone else's work.
+- Write a style guide.
+- Read other's technical writing attentively.
 
 ### Challenge
 
-Apprentices can try to do this other thing. Ideally, they will be challenged to connect what they've learned to some previous knowledge or additional research.
-
+- Write own blogs.
```
Comment (alodahl) [inline roles-in-tech/technical-writing.md]: Write your own blog
Organisation -> organization

File: roles-in-tech/technical-writing.md [hunk 7]
```diff
@@ -1,12 +1,8 @@
...
 ### Projected Time
 
-Example: 30-45 minutes
-- Lesson: ? min
-- Guided Practice: ?-? min
-- Independent Practice: ? min
-- Check for Understanding: ?-? min
+About 30 minutes
 
 ### Prerequisites
```
Comment (alodahl) [inline roles-in-tech/technical-writing.md]: Can you replace the example with our outline on writing good docs? 
https://github.com/Techtonica/curriculum/blob/master/reading-and-writing-documentation/documentation.md

**Comment by alodahl** [comment]
Also, don’t forget about the slides!

File: roles-in-tech/technical-writing.md (hunk 7 above)
Comment (pydevsg) [reply roles-in-tech/technical-writing.md]: Sure, @alodahl, why not.
Done!

**Comment by pydevsg** [comment]
All tasks accomplished!
Got a learnt more about Technical Writing, even I am doing one of them 😄

File: roles-in-tech/technical-writing.md (hunk 7 above)
Comment (alodahl) [reply roles-in-tech/technical-writing.md]: Close... could you move that link to line 9 and remove the word "example"?

Comment (alodahl) [reply roles-in-tech/technical-writing.md]: Also, please change the typos pointed out yesterday.

Comment (pydevsg) [reply roles-in-tech/technical-writing.md]: The requested changes are fixed . 😄

File: roles-in-tech/technical-writing.md [hunk 8]
```diff
@@ -1,90 +1,67 @@
...
 
+[Reading & Writing Documentation](https://github.com/Techtonica/curriculum/blob/master/reading-and-writing-documentation/documentation.md)
 Here are links to lessons that should be completed before this lesson:
```
Comment (alodahl) [inline roles-in-tech/technical-writing.md]: Please replace line 12 with the reading & writing documentation link, unless you have a reason to require the Git lesson.

File: roles-in-tech/technical-writing.md (hunk 6 above)
Comment (alodahl) [reply roles-in-tech/technical-writing.md]: 👆🏼

**Comment by alodahl** [comment]
Great work! When fixing a typo, mention which section it’s in in the message so you can tell the difference if you ever need to go back through commits to find a change.

**Comment by pydevsg** [comment]
Thankyou @alodahl , for guiding me in making this PR more appropriate, I will write the typo error name while writing the commit 😄 .

**Pull request merged**